### PR TITLE
[check-cert-file] improve README

### PR DESCRIPTION
**Pull request by a-know**

### CODE REVIEW

File: check-cert-file/README.md [hunk 1]
```diff
@@ -1,12 +1,53 @@
...
+
+First, build this program.
+
+```
+go get github.com/mackerelio/go-check-plugins
+cd $(go env GOPATH)/src/github.com/mackerelio/go-check-plugins/check-cert-file
+go install
+```
+
+Or you can use this program by installing the official Mackerel package. see [Using the official check plugin pack for check monitoring - Mackerel Docs](https://mackerel.io/docs/entry/howto/mackerel-check-plugins).
```
Comment (susisu) [inline check-cert-file/README.md]: [nit]
```suggestion
Or you can use this program by installing the official Mackerel package. See [Using the official check plugin pack for check monitoring - Mackerel Docs](https://mackerel.io/docs/entry/howto/mackerel-check-plugins).
```

File: check-cert-file/README.md [hunk 2]
```diff
@@ -1,12 +1,53 @@
...
+Next, you can execute this program :-)
+
+```
+check-cert-file --file=/path/to/cert.pem --warning=30 --critical=14
+```
+
+
+## Setting for mackerel-agent
+
+If there is no problem in the execution result, add the setting in mackerel-agent.conf .
```
Comment (susisu) [inline check-cert-file/README.md]: [nit]
```suggestion
If there are no problems in the execution result, add a setting in mackerel-agent.conf .
```

File: check-cert-file/README.md (hunk 1 above)
Comment (a-know) [reply check-cert-file/README.md]: Thanks!! Fixed by efb7e55

File: check-cert-file/README.md (hunk 2 above)
Comment (a-know) [reply check-cert-file/README.md]: Thanks!! Fixed by efb7e55

**Pull request merged**